accounts/templates/accounts/login.html

```diff
@@ -5,9 +5,13 @@
     <form action="{% url 'accounts:register' %}" method="post">
       {% csrf_token %}
       <h1>Create Account</h1>
-      <input type="text" placeholder="Name" />
-      <input type="email" placeholder="Email" />
-      <input type="password" placeholder="Password" />
+      {% if signUpError %}
+      <p class="error">{{ signUpError }}</p>
+      {% endif %}
+      <input type="text" placeholder="Username" name="username" required />
+      <input type="email" placeholder="Email" name="email" required />
+      <input type="password" placeholder="Password" name="password1" required />
+      <input type="password" placeholder="Password" name="password2" required />
```

Comment:
ah so we're doing inputs here instead of django forms, that is fair

Reply:
I did not know Django had a special form technique but I can definitely look into it!

Reply:
yeah checkout "forms.py" in 'recommendations' and also https://docs.djangoproject.com/en/4.2/topics/forms/ & https://docs.djangoproject.com/en/4.2/ref/forms/